tests/testsuite/build_script.rs

```diff
@@ -516,38 +491,27 @@ fn custom_build_env_var_rustc_linker_host_target_with_bad_host_config() {
                 target
             ),
         )
-        .file(
-            "build.rs",
-            r#"
-            use std::env;
-
-            fn main() {
-                assert!(env::var("RUSTC_LINKER").unwrap().ends_with("/path/to/target/linker"));
-            }
-            "#,
-        )
+        .file("build.rs", "fn main() {}")
```

Comment:
ditto to above, confirming that this is being deleted because it was never used

Reply:
Yep.